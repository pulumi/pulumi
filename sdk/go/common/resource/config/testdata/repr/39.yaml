my:nested:
  test.Key: value
