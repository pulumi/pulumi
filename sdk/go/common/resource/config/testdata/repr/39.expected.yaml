decrypt:
  my:nested: '{"test.Key":"value"}'
paths:
  my:nested:
    value:
      test.Key: value
    string: '{"test.Key":"value"}'
    redacted: '{"test.Key":"value"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAe/4IbAAEIdGVzdC5LZXkGc3RyaW5nDAcABXZhbHVl
    secure: false
    isObject: true
  my:nested["test.Key"]:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
